guides/common/modules/snip_verification-ca-set-correctly.adoc

```diff
@@ -0,0 +1,9 @@
+.Verification
+* The following command succeeds on your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+curl https://_{foreman-example-com}_
+----
++
+Replace _{foreman-example-com}_ with the hostname of your {ProjectServer}.
```

Comment:
```suggestion
```

Well yes (unless you meant "{SmartProxy}"? I'd drop this. It might even be wrong if hosts are registered to Smart Proxy Servers. I am unsure about this.

I have not found a Smart Proxy API endpoint that does not require authentication. The only alternative I found was `curl -I http://smart-proxy.example.com/pub/` which returns HTTP code 200.

What do you think @ShimShtein ?

Reply:
I will need to defer to @ekohl. 
I **think** once Katello is installed, it creates a reverse-proxy to Foreman server, and maybe we can use it to forward requests from the client through proxy.

Any thoughts on this one?

Reply:
Isn't the point here that `{ProjectServer}` is the CA regardless to what the host is registered?

@maximiliankolb I'm adding this line because I have a bug report from our support that users tend to copy-past without editing the command accordingly as discussed in this PR #3187. I can hide it for orcharhino or include it only for Satellite if you wish.

Reply:
Maybe you're right about the CA. I am OK keeping it for all build targets.

Reply:
Actually, if the Smart Proxy uses certificates that were issued by the Foreman server (and are validated by the same CA), the user can also access `https://smart.proxy.example.com:9090/features` (a proxy endpoint). If the CA is not set on the host, the communication will fail, since the host won't be able to recognize the certificate presented by the Smart Proxy.

Reply:
@ShimShtein Does it matter whether the user uses the hostname of ProjectServer or SmartProxyServer for the verification? Or does he have to check both?

Reply:
It depends on user's network configuration. In some use cases, hosts have only access to the proxy, in others - the hosts have direct connection to the server.